code/sorting/src/insertion_sort/insertion_sort.c

```diff
@@ -15,7 +15,7 @@ int main()
 	insertion(arr, n);
 	return 0;
 }
-
+/* Function to sort an array using insertion sort*/
```

Comment:
This comment isn't really needed.

Reply:
Instead of this comment, you can consider renaming the function to `insertionSort`.